frontend/src/components/editor/chrome/panels/packages-panel.tsx

```diff
@@ -599,7 +606,11 @@ const DependencyTreeNode: React.FC<{
         {isTopLevel && (
           <div className="flex gap-1 invisible group-hover:visible">
             <UpgradeButton packageName={node.name} onSuccess={onSuccess} />
-            <RemoveButton packageName={node.name} onSuccess={onSuccess} />
+            <RemoveButton
+              packageName={node.name}
+              tags={node.tags as { kind: string; value: string }[]}
```

Comment:
Ideally we should update our generated types for `node.tags` to be more type safe (instead of `Record<string,any>`). I can make a follow up PR.